cwl/workflow/data_assembly_DNA_wf.cwl

```diff
@@ -0,0 +1,91 @@
+cwlVersion: v1.0
+class: Workflow
+id: data_assembly_DNA_wf
+requirements:
+  - class: MultipleInputFeatureRequirement
+  - class: SubworkflowFeatureRequirement
```

Comment:
When run on cwltool with no args, I get the error:
```
ERROR Tool definition failed initialization:
Workflow contains scatter but ScatterFeatureRequirement not in requirements
```

Reply:
fixed